crate_universe/defs.bzl

```diff
@@ -236,78 +235,55 @@ A rule for downloading Rust dependencies (crates).
 Environment Variables:
 - `REPIN`: Re-pin the lockfile if set (useful for repinning deps from multiple rulesets).
 - `RULES_RUST_REPIN`: Re-pin the lockfile if set (useful for only repinning Rust deps).
-- `RULES_RUST_CRATE_UNIVERSE_RESOLVER_URL_OVERRIDE`: Override URL to use to download resolver binary - for local paths use a `file://` URL.
-
-`override` Example:
-
-```python
-load("@rules_rust//crate_universe:defs.bzl", "crate_universe", "crate")
-
-crate_universe(
-    name = "mylib",
-    # [...]
-    overrides = {
-        "tokio": crate.override(
-            extra_rust_env_vars = {
-                "MY_ENV_VAR": "MY_ENV_VALUE",
-            },
-            extra_build_script_env_vars = {
-                "MY_BUILD_SCRIPT_ENV_VAR": "MY_ENV_VALUE",
-            },
-            extra_bazel_deps = {
-                # Extra dependencies are per target. They are additive.
-                "cfg(unix)": ["@somerepo//:foo"],  # cfg() predicate.
-                "x86_64-apple-darwin": ["@somerepo//:bar"],  # Specific triple.
-                "cfg(all())": ["@somerepo//:baz"],  # Applies to all targets ("regular dependency").
-            },
-            extra_build_script_bazel_deps = {
-                # Extra dependencies are per target. They are additive.
-                "cfg(unix)": ["@buildscriptdep//:foo"],
-                "x86_64-apple-darwin": ["@buildscriptdep//:bar"],
-                "cfg(all())": ["@buildscriptdep//:baz"],
-            },
-            extra_bazel_data_deps = {
-                # ...
-            },
-            extra_build_script_bazel_data_deps = {
-                # ...
-            },
-        ),
-    },
-)
-```
+- `RULES_RUST_CRATE_UNIVERSE_RESOLVER_URL_OVERRIDE`: Override URL to use to download resolver binary 
```

Comment:
Can you add a note at the top of this docstring highlighting that this rule is experimental and subject to breaking changes at any time?

Reply:
I can add that here, __or__, I can do so in a header template after https://github.com/bazelbuild/rules_rust/pull/706 what do you think?

Reply:
Ah, didn't think about flatten.md. Lame but makes sense

Reply:
Updated